content/manuals/build/bake/remote-definition.md

```diff
@@ -158,6 +134,12 @@ target "default" {
 }
 ```
 
+The following example uses `-f` to specify two Bake definitions:
+
+- `-f bake.hcl`: this definition is loaded relative to the Git URL.
+- `-f cwd://local.hcl`: this definition is loaded relative to the current
+  working directory where the Bake command is executed.
```

Comment:
related to this, we have this case in metadata-action: https://github.com/docker/metadata-action/blob/70b2cdc6480c1a8b86edf1777157f8f437de2166/README.md?plain=1#L282-L297

Reply:
Updated to call out this example